project/Dependencies.scala

```diff
@@ -29,6 +29,7 @@ object Dependencies {
     "xml-resolver" % "xml-resolver" % "1.2",
     "commons-io" % "commons-io" % "2.6",
     "jline" % "jline" % "2.14.6",
+    "com.typesafe" % "config" % "1.4.0"
```

Comment:
Using Config as mentioned before to support configuring validators in an abstract way.  Validators being a completely unknown and non-standardized process makes us support arbitrary configuration.  To support definition and access of the arbitrary configurations in a standardized way we use this.